inspirehep/modules/workflows/workflows/article.py

```diff
@@ -392,11 +394,32 @@
 ]
 
 
-MARK_IF_UPDATE = [
+HALT_FOR_MATCH_APPROVAL_OR_MARK_IF_UPDATE = [
     IF_ELSE(
-        article_exists,
-        mark('is-update', True),
-        mark('is-update', False),
+        exact_match,
+        [
+            mark('is-update', True),
+            mark('exact-matched', True),
+        ],
+        IF_ELSE(
+            fuzzy_match,
+            [
+
```

Comment:
extra newline

Reply:
If you added it to make it more readable, I suggest using a new array var instead (the equivalent of moving to a new function).

Reply:
just extra